oidc-api/src/test/java/uk/gov/di/authentication/oidc/contract/AccountInterventionServiceTest.java

```diff
@@ -358,12 +358,7 @@ private static DslPart createAisResponseBody(
                             body.object(
                                     "intervention",
                                     (obj) -> {
-                                        obj.numberType("updatedAt", TIME_NOW);
                                         obj.numberType("appliedAt", TIME_NOW);
-                                        obj.numberType("sentAt", TIME_NOW);
-                                        obj.stringType("description", "TEST_DESCRIPTION");
-                                        obj.numberType("reprovedIdentityAt", TIME_NOW);
-                                        obj.numberType("resetPasswordAt", TIME_NOW);
                                     });
```

Comment:
Is the default to return only "appliedAt". Do we care about the other fields at any point in the `intervention` object ?

Reply:
No the default is to return either all the fields or all the fields excluding `reprovedIdentityAt` and `resetPasswordAt`. The only field we use is `appliedAt` and we don't care about any other fields

Reply:
Cool - this is fine